src/modules/TurboAdmin.sol

```diff
@@ -7,30 +7,72 @@ import {Auth, Authority} from "solmate/auth/Auth.sol";
 import {TurboBooster} from "./TurboBooster.sol";
 import {TurboClerk} from "./TurboClerk.sol";
 
-interface IMasterOracle {
-    function add(address[] calldata underlyings, address[] calldata _oracles)
-        external;
-
-    function changeAdmin(address newAdmin) external;
-
-    function admin() external view returns (address);
-}
-
+/** 
+ @title Turbo Admin of Turbo Fuse Pool and Turbo Timelock
+*/
 contract TurboAdmin is Auth {
-
+    /// @notice generic error thrown when comptroller call fails
     error ComptrollerError();
 
-    /// @notice the fuse comptroller
+    /// @notice the fuse comptroller associated with the TurboAdmin
     Comptroller public immutable comptroller;
 
+    /// @notice the turbo timelock
     TimelockController public immutable timelock;
 
+    /// @notice constant zero interest rate model
+    address public constant ZERO_IRM = 0xC9dB5A1034BcBcca3f59dD61dbeE31b78CeFD126;
+
+    /// @notice cToken implementation
+    address public constant CTOKEN_IMPL = 0x67Db14E73C2Dce786B5bbBfa4D010dEab4BBFCF9;
+
     /// @param _comptroller the fuse comptroller
     constructor(Comptroller _comptroller, TimelockController _timelock, Authority _authority) Auth(address(0), _authority) {
         comptroller = _comptroller;
         timelock = _timelock;
     }
 
+    // ************ TURBO ADMIN FUNCTIONS ************
+
+    function addCollateral(
```

Comment:
i would just make the admin allow delegatecalling arbitrary contracts so u dont need to expictly define upfront the functions that can be called

Reply:
like the makerdao pause proxy (core governance contract) https://github.com/dapphub/ds-pause/blob/0763eafcf926fd2e073aee5f047f3decb842231c/src/pause.sol#L92

Reply:
well i guess u get the same kinda functionality with schedule()